src/components/BookComponent/AddBook.module.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 10px 30px;
+  border: 2px solid rgb(226, 226, 226);
+  margin: 10px 22px;
+}
+
+.Title {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.container form {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container form > input:nth-child(1) {
+  width: 23rem;
+  height: 30px;
+  border-radius: 4px;
```

Comment:
- [ ] Please kindly ensure that your design meets the project design requirements(colors, font, padding, button, spacing etc)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/179132474-999d8696-338a-4a3c-bcf6-bb3ee16394f6.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/179132428-e2352e3f-9b7e-4137-93a0-ac48d533303d.png)

Reply:
@AnselemOdims Thank you, I have fixed this up but there is a difference in the author name which is not on the wireframe.